### PR TITLE
Add SGE Network 

### PR DESCRIPTION
**Pull request by shang-chi-7**
This PR adds the SGE Network chain to the chain registry.

### CODE REVIEW

File: testnets/sgenetworktestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sge-network-3",
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/assetlist.json:3]: please replace chain_id with chain_name (as sgenetworktestnet)

File: testnets/sgenetworktestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,122 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "Sge Network Testnet",
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/chain.json:3]: should be sgenetworktestnet without spaces--should match the directory name

File: testnets/sgenetworktestnet/assetlist.json (hunk 1 above)
Comment (shang-chi-7) [reply testnets/sgenetworktestnet/assetlist.json:3]: Our chain id is sge-network-3, shall we keep the folder name as sge-network-3 ?

File: testnets/sgenetworktestnet/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-1015",
+        "source_denom": "usge"
+      },
+      "logo_URIs": {
+        "png": "https://sgenetwork.io/images/sgenetwork-logo.png",
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/assetlist.json:53]: please point these to the local /testnets/sgenetworktestnet/images/ folder in this repo, and upload any necessary files.

File: testnets/sgenetworktestnet/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        {
+          "denom": "usge",
+          "exponent": 6
+        }
+      ],
+      "base":"ibc/CFDF212C4BE8349AEDF9DE5006F394717C7E9E814742E1D6DE95AE288F58054C",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "ibc": {
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/assetlist.json:47]: we don't use this property anymore (we use 'traces'). What is this ibc transfer describing?

File: testnets/sgenetworktestnet/assetlist.json (hunk 1 above)
Comment (reeves89) [reply testnets/sgenetworktestnet/assetlist.json:3]: @JeremyParish69 Got your point, will do that. cc @shang-chi-7

File: testnets/sgenetworktestnet/assetlist.json (hunk 4 above)
Comment (reeves89) [reply testnets/sgenetworktestnet/assetlist.json:47]: @JeremyParish69
Those are added thinking, transferring tokens from keplr to osmosis or osmosis to keplr?  - is it wrong? if yes,  can you please share more info about it and what should we do?

File: testnets/sgenetworktestnet/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply testnets/sgenetworktestnet/assetlist.json:3]: no, the folder name sgenetworktestnet or saagetestnet would be most appropriate. The chain_id only goes in the chain.json file as `chain_id`.

File: testnets/sgenetworktestnet/assetlist.json (hunk 4 above)
Comment (JeremyParish69) [reply testnets/sgenetworktestnet/assetlist.json:47]: oh, so this is describing the transition from sgetestnet to osmosistestnet? If that's the case, it should not be listed here. I'd remove the ibc asset entirely.

**Comment by shang-chi-7** [comment]
@JeremyParish69 Can you please review.

File: testnets/sgenetworktestnet/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "usge",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "logo_URIs": {
+        "png": "./images/logo.png",
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/assetlist.json:24]: please reference the full url to this repo, e.g., "https://raw.githubusercontent.com/cosmos/chain-registry/master/sgenetworktestnet/images/logo.png"

File: testnets/sgenetworktestnet/assetlist.json [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        "svg": "./images/logo.svg"
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/assetlist.json:25]: for the token logos, please upload images with square dimensions. The logotypes you've uploaded are fine for Chain images, but not for Asset images.

File: testnets/sgenetworktestnet/chain.json [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+      },
+      {
+        "id": "58556b5fb572e20d41ce686149ab7b1646ad63a9",
+        "address": "65.108.15.170:26656",
+        "provider": "ramuchi.tech"
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "./images/logo.png",
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/chain.json:78]: please use full URL

File: testnets/sgenetworktestnet/assetlist.json (hunk 6 above)
Comment (JeremyParish69) [reply testnets/sgenetworktestnet/assetlist.json]: For the chain logos, those logos appropriate. But for the asset logos, yes you technically made the files square, but we don't want the entire logotype with the wording 'Sports Gaming and Entertainment Network', but only the logo asset--the golden 'S' thing--which would be appropriate as an actual token design.

Comment (shang-chi-7) [reply testnets/sgenetworktestnet/assetlist.json]: @JeremyParish69 Please review now.

Comment (JeremyParish69) [reply testnets/sgenetworktestnet/assetlist.json]: Thanks for this. The SVG token logo is a rather poor quality conversion from the png. Could we either remove it and only use the png, or else grab the icon from the 'logo.svg' image (which doesn't have the subtle reflections in it) and use that for both?

Comment (shang-chi-7) [reply testnets/sgenetworktestnet/assetlist.json]: @JeremyParish69 Updated the PR

File: testnets/sgenetworktestnet/assetlist.json [hunk 8]
```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sgenetworktestnet",
```
Comment (JeremyParish69) [inline testnets/sgenetworktestnet/assetlist.json:3]: checks failed. This property should be "chain_name": "sgenetworktestnet", not the ID

**Comment by JeremyParish69** [comment]
Apologies, there's already a saagetestnet chain, which I believe is the same thing?? Could we bring any updates over to that chain instead, please?

**Comment by shang-chi-7** [comment]
> sgenetworktestnet

@JeremyParish69 Shall we remove the SaageTestnet ?

**Comment by JeremyParish69** [comment]
> > sgenetworktestnet
> 
> @JeremyParish69 Shall we remove the SaageTestnet ?

I guess changing it is OK since it's only the testnet registered, although I'd like to ask, why not stick with Saage? Did the network rebrand?

**Comment by reeves89** [comment]
> > > sgenetworktestnet
> > 
> > 
> > @JeremyParish69 Shall we remove the SaageTestnet ?
> 
> I guess changing it is OK since it's only the testnet registered, although I'd like to ask, why not stick with Saage? Did the network rebrand?

 Did the network rebrand?
Yes.

**Pull request merged**